example/README.md

```diff
@@ -0,0 +1,48 @@
+## WARNING: This is Alpha software and not intended for use until a stable release.
+
+# Running M3DB on GCP
+
+Setup GCP for m3db:
+
+    1. Download necessary packages
+        $ sudo apt-get install golang golint make git golang-glide
+    2. Configure git
+        $ git config --global user.email "test@example.com"
+        $ git config --global user.name "First Last"
+    3. Set GOPATH:
+        $ export GOPATH=/home/<user_name>/code
+    4. Create m3db directory
+        $ mkdir -p /home/<user_name>/code/src/github.com/m3db
+    5. cd into m3db directory and git clone m3db
+        $ git clone https://github.com/m3db/m3db
+    6. Checkout `braskin/static_topology` (until this is landed in master)
+        $ git checkout origin/braskin/static_topology
+    7. Build m3db
+        $ git submodule update --init --recursive
+        $ glide install
+        $ make services
+    8. There are three config files in this directory (`config1.yaml`, `config2.yaml`, `config3.yaml`) - you will use one config per m3db node
```

Comment:
naming nit: would be better to include service name, and the fact that it's for a different server in the filename. e.g. `m3dbnode-server1-config.yaml`

Reply:
'one config per m3db node' is a little vague, might be clearer to say per GCP instance?

Reply:
Done